python/axipcie/_AxiPcieCore.py

```diff
@@ -147,25 +148,43 @@ def __init__(self,
                     XIL_DEVICE_G = 'ULTRASCALE'
                     qsfpOffset = [0x74_000,0x71_000]
 
-                    for i in range(2):
-                        self.add(xceiver.Qsfp(
-                            name    = f'Qsfp[{i}]',
-                            offset  = qsfpOffset[i],
-                            memBase = self.AxilBridge.proxy,
-                            enabled = False, # enabled=False because I2C are slow transactions and might "log jam" register transaction pipeline
-                        ))
+                    if not useSfp:
```

Comment:
@mmishra9 useSFP should be a list of booleans (instead of a single boolean) in case you get a mix of SFP and QSFP on the front panel.